### ISSUE TITLE
Infographics not loading on Chrome Android 

### ISSUE THREAD

**Issue by setcz** [faktaoklimatu/web-core#106]
(Already posted also on Slack)

On Chrome, Android, the web doesn't show infographics when going to the page. It s shown only after refresh of the page. See screengrab below:

https://user-images.githubusercontent.com/10813593/128175985-8b67fabf-d391-4a09-9076-c90931a3be36.mp4

System and browser information:
![Screenshot_20210803-185432](https://user-images.githubusercontent.com/10813593/128176128-c74835b2-b869-451e-8aa6-a74860ce5a5b.png)

**Comment by setcz** [comment]
Can someone please confirm whether this is happening on other android phones as well?

If so, I consider this to be an important bug, which should be investigated - as for mobile users it is removing the main part of the page 

See one more screen grab of the whole flow:
Uploading screen-20210813-090904.mp4…

**Comment by mukrop** [comment]
As mentioned previously, no such problem on my device (also Android 11, also Chrome 92.0.4515.131). Although that does not yet mean nobody else has it, we're still not able to reproduce it.

**Comment by mgrabovsky** [comment]
I'm unable to reproduce this in Chrome on Android 6.

**Comment by jankrcal** [comment]
I could not repro either.

**Comment by setcz** [comment]
Thanks all for checking! I am much less worried now.

I tested it on Opera Mini (on the same phone) - and the infographics are loading there just fine. 

However, the menu is super broken there:

![Screenshot_20210815-160623](https://user-images.githubusercontent.com/10813593/129481283-0c42da0d-aa1b-49f7-b37f-1c3d30b59cc8.png)

This is the Opera Mini I am using:

![Screenshot_20210815-160706](https://user-images.githubusercontent.com/10813593/129481297-8bc968ec-468b-41cd-9599-3871fffa04aa.png)

(This should be probably a separate bug, however I'd like to test it on some other phone first)

**Comment by setcz** [comment]
https://user-images.githubusercontent.com/10813593/141674388-5f8540cb-b11a-4781-95b0-ed5f622a5218.mp4

Ha, I managed  to reproduce the  infographic not showing on another device! Samsung Galaxy tab S6, Chrome. See the recording above.

I still consider this to be quite severe bug, as if users are not aware that they need to reload, they will miss the main content of the page.

**Comment by setcz** [comment]
@mukrop wanted me to try to debug this myself - I managed to use the DevTools on desktop to look at the connected Android device. 

I may have found something useful. This is what shows whenever I load a page for the first time (and the infographic is not showing):
<img width="1294" alt="Screenshot 2021-11-14 at 21 56 13" src="https://user-images.githubusercontent.com/10813593/141698440-e9cd302a-931b-454a-89b0-72ea9643e18e.png">

This yellow warning disappears when I reload the page - and the infographics is actually shown:
<img width="1292" alt="Screenshot 2021-11-14 at 21 56 27" src="https://user-images.githubusercontent.com/10813593/141698471-23ffbca6-8cc8-42c7-8159-3d9070dc25ff.png">

The error message in the first image reads: "[Intervention] Images loaded lazily and replaced with placeholders. Load events are deferred. See https://crbug.com/954323 "

And this page is linked from that page - https://www.chromestatus.com/feature/5645767347798016 

Could this be it? Could someone who knows how this works please try to experiment with adding eager loading to the main infographic? Thanks!

**Comment by jankrcal** [comment]
Nice! 

If that works, that's obviously a bug in Blink, somehow not detecting well that the image is within the viewport. I'd love this to be debugged a bit deeper so that we/I can file a crbug. What's fishy that it triggers rarely, just on a few devices (of @setcz).

**Comment by mukrop** [comment]
That debugging indeed helps a lot!

1) Cross-check: @setcz, do you have the "Lite mode" enabled? (See [Chrome documentation](https://support.google.com/chrome/answer/2392284?hl=en&co=GENIE.Platform%3DAndroid) if unsure.)
If so, can you disable it and try again?

2) (Independent of 1.) Can you check if you have the same issue on [this development build](https://cz-fakta-o-klimatu--preview-lazy-loading-bug-ichhlehv.web.app/infografiky/akteri-klimatickych-jednani)?

@jankrcal: I suspect it may be caused by the image dimensions setup. In order not to repeat the title, sources and license information, the infographics are displayed larger than the box they are displayed in a precise manner to cut the header, footer and extra margins. I can imagine the setup is too sophisticated for the lazy loading heuristic to realize the majority of the image is actually displayed...

**Comment by setcz** [comment]
1) Yes, I do have the lite mode enabled. I tried also in the incognito mode now, and it works just fine (according to this [help centre article](https://support.google.com/chrome/answer/2392284?hl=en&co=GENIE.Platform%3DAndroid
): "If you browse in private, Lite mode won’t work" - therefore I suspect that this might be it)

2) This seems to work just fine!

**Comment by mukrop** [comment]
Nice, thanks.

The bug then stems from Chrome-side proactive optimizations in Lite mode, most likely due to (intentional) partial hiding of the image. The bugfix is setting the load mode to `eager` in the case of infographics images.

@jankrcal, do you want to investigate more and/or file an issue?

**Comment by jankrcal** [comment]
I think I have enough input, thanks!

Lite mode was the missing bit, I can even repro myself.
